LTMorphingLabel/LTMorphingLabel.swift

```diff
@@ -560,3 +560,10 @@ extension LTMorphingLabel {
     }
 
 }
+
+extension LTMorphingLabel {
+    public override var intrinsicContentSize: CGSize {
```

Comment:
is it save to do this globally for all `LTMorphingLabel`? This would require us to test all usage of `LTMorphingLabel` in lots of different configurations.
It thought you only wanted to open `newRects` so that the custom subclass can use it

Reply:
we only really use `LTMorphingLabel` in one place actually and we subclassed it there to override `intrinsicContentSize`

Reply:
ok, fair enough!